File: src/TaskJava/Occurences.java
package TaskJava;



public class Occurences {
    public static void main(String[] args) {
        String word = "Mississippi";
        int Icounter = 0;
        int Scounter = 0;
        for (int i = 0; i < word.length(); i++) {
            if (word.charAt(i) == 'i'){
                Icounter++;
            }
            else if (word.charAt(i) == 's'){
                Scounter++;
            }
        }
              System.out.printf("%d%n%d", Icounter, Scounter);

    }
}

File: src/TaskPython/occurences.py
word = 'Mississippi'
counter_for_I = 0
counter_for_S = 0
for i in word:
    if 'i' in i:
        counter_for_I += 1
    elif 's' in i:
        counter_for_S += 1
print("the number of i is ", counter_for_I)
print("the number of s is ", counter_for_S)
